deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Curious, what is the reason for the version bump?

Reply:
I didn't notice I used different `controller-gen` version. Fixed it. Thank you.